lib/fluent/formatter.rb

```diff
@@ -131,8 +131,22 @@ class JSONFormatter < Formatter
       include HandleTagAndTimeMixin
       include StructuredFormatMixin
 
+      config_param :disable_oj, :bool, :default => false
```

Comment:
Option like `json_parser oj` or `json_parser yajl` looks better because there exists more parsers (cf. https://github.com/intridea/multi_json#supported-json-engines)